scripts/travis/travis-tests

```diff
@@ -19,6 +19,35 @@ if [ "$UNAME_M" != "x86_64" ]; then
 	SKIP_TRAVIS_TEST=1
 fi
 
+install_retry_counter=0
+max_apt_retries=10
+
+# This function loops a couple of times over apt-get, hoping to
+# avoid CI errors due to errors during apt-get
+# hashsum mismatches, DNS errors and similar things
+apt_install () {
+        set +e
+        while true; do
+                install_retry_counter=$(expr $install_retry_counter + 1)
```

Comment:
let  install_retry_counter++

Reply:
`let` needs `/bin/bash`. Initially I had `let` but had to change it to `expr`. We can, of course, change the whole script to `/bin/bash`.

Reply:
> let install_retry_counter++

Won't work on `dash` which might be used as `/bin/sh` on Ubuntu.
Maybe we want to make travis-tests to explicitly use bash and then `let cnt++` or `((cnt++))` would work.

```
diff --git a/scripts/travis/travis-tests b/scripts/travis/travis-tests
index e9c2d5c6a..d159bb71f 100755
--- a/scripts/travis/travis-tests
+++ b/scripts/travis/travis-tests
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 set -x -e
 
 TRAVIS_PKGS="protobuf-c-compiler libprotobuf-c-dev libaio-dev python-future
```

Reply:
I vote to force using bash for all our shell scripts, but @adrianreber it is up to you

Reply:
switched to `let` and bash